nap/fields.py

```diff
@@ -153,6 +161,26 @@ def descrub_value(self, val, for_read=False):
         return obj_list
 
 
+class SimpleListField(Field):
+    """A list of simple types (e.g. strings, not Resources)"""
+    def scrub_value(self, val):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
+
+    def descrub_value(self, val, for_read=False):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
```

Comment:
Will the field be `None` if you instantiate a new version via constructor before saving? As in:

```
>>> r = MyResource()
>>> r.my_simple_list
None
```

If so, perhaps a more robust way to handle `None` is to override `__init__` and pass `default=[]` to `super's __init__` if no default was provided.

But I may also be wrong and the fields all get descrubbed on construction? I haven't actually tried it.